wsd/COOLWSD.cpp

```diff
@@ -3482,6 +3483,9 @@ class PrisonerRequestDispatcher final : public WebSocketHandler
 
         // Remove consumed data.
         data.eraseFirst(read);
+#else
+        Poco::Net::HTTPRequest request;
```

Comment:
Why not have the `http::Request request;` on line 3468 outside the `#if !MOBILEAPP` directive, two lines above, and avoid this? Why not have `request` of type `http::Request`? (We are reducing Poco dependency.)